### DISCUSSION TITLE
2023-07-12 Data Architecture Meeting

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_mecha#216] [Meeting Agendas]
- Metadata for info used to identify animals (e.g. tattooing, bleaching, tailpainting, etc.)

**Comment by ttngu207** [comment]
Final architecture for the SLEAP reader is unconfirmed. We are considering the possibility of modifying the reader so that model path does not have to be specified, but instead extracted from metadata.

See PR: #206
